=== modified file 'swift/obj/replicator.py'
--- swift/obj/replicator.py     2011-04-20 22:43:11 +0000
+++ swift/obj/replicator.py     2011-05-12 19:19:03 +0000
@@ -271,9 +271,9 @@
             '--contimeout=%s' % self.rsync_io_timeout,
         ]
         if self.vm_test_mode:
-            rsync_module = '%s::object%s' % (node['ip'], node['port'])
+            rsync_module = '[%s]::object%s' % (node['ip'], node['port'])
         else:
-            rsync_module = '%s::object' % node['ip']
+            rsync_module = '[%s]::object' % node['ip']
         had_any = False
         for suffix in suffixes:
             spath = join(job['path'], suffix)
